docs/_sass/footer.scss

```diff
@@ -288,7 +290,7 @@ footer {
   .footer-columns {
     display: flex;
     justify-content: center;
-    // padding-right: 37px;
+    // padding-right: 43px;
```

Comment:
@Anita-ihuman, this change is not affecting the final styling. It can be removed.

Reply:
I have fixed this